README.md

```diff
@@ -1 +1,22 @@
-# nimble-golang-sdk
\ No newline at end of file
+# HPE Nimble Storage SDK in Go Programming Language
+This is a Go Software Development Kit (SDK) for [HPE Nimble Storage](http://hpe.com/storage/nimblestorage) arrays. The HPE Nimble Storage array has a Representational State Transfer (REST) web service application programming interface (API). The SDK implements a simple client Go modules for communicating with the HPE Nimble Storage REST API. The Go module is being used to communicate with the API over HTTPS.
+
+The SDK provides a Go modules to interact with the HPE Nimble Storage REST API. The code abstracts the lower-level API calls into Go objects that you can easily incorporate into any automation or DevOps workflows. Use it to create, modify and delete most resources like volumes, volume collections, initiator groups and more, as well as perform other tasks like snapshotting, cloning, restoring data, etc.
```

Comment:
Remove the word "a" in the first sentence?

It is worth mentioning get/read operations in the third sentence.

Reply:
I will remove "a" . Please note that this is the initial README. It will be rewritten by Michael.